src/Ethereum/ABI/Function.cpp

```diff
@@ -11,6 +11,8 @@
 
 namespace TW::Ethereum::ABI {
```

Comment:
this file is not deleted because we need to keep it?

Reply:
`Ethereum::ABI::Function` is used in `TWEthereumAbiEncode`, `TWEthereumAbiDecodeOutput`, `TWEthereumAbiFunctionGetType`, `TWEthereumAbiFunctionAddParam*` functions. It's designed the way to forward the function calls to Rust